### PR TITLE
Angular Developer's Guide to Their First React App

### PR DESCRIPTION
**Pull request by michael-martin-al**

### CODE REVIEW

File: posts/2015-12-29-an-angular-developers-first-react-app.md [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+That's it for the component definition. React components have a `props` property with key/value pairs for all attributes passed to them through attributes in JSX. To use this component, we just need to supply the value of the `title` property like this:
+
+```javascript
+<Title title={myTitle} />
+```
+
+Notice that in JSX, you don't need any single or double quotations surrounding your property value. You just enclose the `myTitle` variable with a single set of curly braces. Hopefully, this feels a lot like Angular's double curly brace syntax for binding to `$scope` variables. Just remember, no quotes.
+
+
+#### Statless Immutable Glory
```
Comment (zackify) [inline posts/2015-12-29-an-angular-developers-first-react-app.md]: think you could proof read and fix up a few of these little mistakes? Then I'll merge it for you :)

Comment (michael-martin-al) [inline posts/2015-12-29-an-angular-developers-first-react-app.md]: Sure. Let me look at your feedback and I'll make a new commit.

**Comment by michael-martin-al** [comment]
@zackify anything else I need to change?

**Comment by zackify** [comment]
Looks good! Merging!

**Comment by michael-martin-al** [comment]
@zackify thanks for merging! I did notice a problem with the live post. It seems the article is not inside the post layout. The only thing that shows up when I browse to the article is the compiled markdown. It is not inside HTML/BODY tags.

**Comment by zackify** [comment]
@michael-martin-al https://github.com/Legitcode/ReactJSNews/commit/a0c985d772759c7aa208a52d4f4d8ee4697c40ee found the problem. Fixed it. Think you could post something that isn't a repost next time by the way? I'd like to have more original content, but I understand either way. Thanks for your posts! :D

**Comment by michael-martin-al** [comment]
Absolutely. Any request for specific topics?
On Jan 4, 2016 7:10 PM, Zach Silveira <notifications@github.com> wrote:

@michael-martin-al<https://github.com/michael-martin-al> a0c985d<https://github.com/Legitcode/ReactJSNews/commit/a0c985d772759c7aa208a52d4f4d8ee4697c40ee> found the problem. Fixed it. Think you could post something that isn't a repost next time by the way? I'd like to have more original content, but I understand either way. Thanks for your posts! :D

**Comment by zackify** [comment]
react-router + dynamic routing + server rendering. I was going to write on
that myself, been so busy though. Maybe on how distributing components too.

**Pull request merged**